Remotion/Data/DomainObjects.UnitTests/TestDomain/Order.cs

```diff
@@ -46,6 +48,7 @@ public int RedirectedRedirectedOrderNumber
       get { return RedirectedOrderNumber; }
     }
 
+    [DateTimeStorageType(DateTimeStorageType.DateTime)]
     public abstract DateTime DeliveryDate { get; set; }
```

Comment:
Somehting I didn't see before: we only have a not-nullable property for the integration tests. We should also do a nullable one for reading and writing. Not sure where we got one that we can tweak. If you want to add one here, I don't know how big the impact would be in terms of cleanup in the sql scipts etc.